tools/python/register_custom_ops_pytorch_exporter.py

```diff
@@ -34,6 +35,21 @@ def tril(g, self, diagonal):
     register_custom_op_symbolic('::triu', triu, _onnx_opset_version)
     register_custom_op_symbolic('::tril', tril, _onnx_opset_version)
 
+    if is_ortmodule:
```

Comment:
Is there any reason to not always register this op? Maybe we dont want that for inference? If that is the case, "is_training" would be more meaningfull than "is_ortmodule".